frontend/lib/map/map/map_with_futures.dart

```diff
@@ -30,13 +32,13 @@ class MapWithFutures extends StatelessWidget {
         final position = snapshot.data;
 
         return MapContainer(
-          onFeatureClick: onFeatureClick,
-          onNoFeatureClick: onNoFeatureClick,
-          locationAvailable: position?.isAvailable() ?? false,
-          userLocation: position?.toLatLng(),
-          onFeatureClickLayerFilter: onFeatureClickLayerFilter,
-          onMapCreated: onMapCreated,
-        );
+            onFeatureClick: onFeatureClick,
```

Comment:
Please undo the intendation change or is there a reason for it?

Reply:
Sadly had to redo, otherwise checks are failing :/